framework/src/modules/dpos_v2/commands/delegate_registration.ts

```diff
@@ -113,15 +146,19 @@ export class DelegateRegistrationCommand extends BaseCommand {
 			isBanned: false,
 			pomHeights: [],
 			consecutiveMissedBlocks: 0,
-			// TODO: Issue #7665
-			commission: 0,
-			lastCommissionIncreaseHeight: 0,
-			sharingCoefficients: [{ tokenID: Buffer.alloc(8), coefficient: Buffer.alloc(24) }],
+			commission: 10000,
```

Comment:
maybe define constant?

Reply:
A local constant, or a constant in default config, and then get the value from config, like for `DELEGATE_REGISTRATION_FEE`?

Reply:
Probably both are valid for this, but i think constant like `DELEGATE_REGISTRATION_FEE ` is fine for now